plugin/entryAttributes.go

```diff
@@ -179,6 +180,15 @@ func (a *EntryAttributes) SetSize(size uint64) *EntryAttributes {
 	return a
 }
 
+// Inaccessible returns true if accessing an entry is expected to fail due to missing permissions.
+func (a *EntryAttributes) Inaccessible() bool {
```

Comment:
I think this should move over to the `EntryBase` class. Main reason is b/c the `inaccessible` attribute isn't something that you're going to filter on.

Reply:
Making it an attribute made it straight-forward to set from an external plugin.

Reply:
I'll look into how the external plugin should work.

Reply:
Added external plugin support.